ci-operator/step-registry/cucushift/installer/check/aws/private/cucushift-installer-check-aws-private-commands.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REGION="${LEASED_RESOURCE}"
+INFRA_ID=$(jq -r '.infraID' ${SHARED_DIR}/metadata.json)
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+ret=0
+
+#Check there's no external load balancer created in this cluster
+
+EXT_LB=$(aws --region ${REGION} elbv2 describe-load-balancers |jq --arg var "${INFRA_ID}" -r '.LoadBalancers[] | select(.LoadBalancerName == "\($var)-ext")')
```

Comment:
To be more accurate, I guess we can try to use tag name - `kubernetes.io/cluster/<INFRA_ID>` to filter out all the lb, then check them one by one, to ensure there is no `Internet-facing` lb created.